### ISSUE TITLE
Sort imports and remove unused imports

### ISSUE THREAD

**Issue by tbroyer** [coveo/fmt-maven-plugin#11]
The `Formatter` class won't sort imports or remove unused imports. This can be done using the `ImportOrderer` and `RemoveUnusedImports` classes respectively.

See https://github.com/google/google-java-format/pull/96 which adds this as a default behavior in the `Formatter` class.

(note that `java -jar google-java-format-1.3-all-deps.jar` uses `FormatFileCallable` instead of directly calling the `Formatter`; `FormatFileCallable` sort and cleans up imports in addition to formatting the code; actually, this is probably what this plugin should use too)

**Comment by jbduncan** [comment]
For [Spotless](https://github.com/diffplug/spotless/), we effectively call the following lines one after another.

```java
String source = ...;
String formatted = new Formatter().formatSource(...);
formatted = RemoveUnusedImports.removeUnusedImports(formatted);
formatted = ImportOrderer.reorderImports(formatted);
```

Using `FormatFileCallable` would be tricky if not impossible without reflection AFAICT, because it requires an instance of `CommandLineOptions`, which is a default-visibility class.

**Comment by jbduncan** [comment]
On a somewhat different note, I'm interested in seeing this issue resolved as well, as it would make implementing https://github.com/jrtom/jung/issues/86 easier for us.

**Comment by pastjean** [comment]
Fixed by #13

**Comment by jbduncan** [comment]
Great to see that this issue is resolved now. Thank you very much @pastjean and @therealppa. :+1:
